examples/index.html

```diff
@@ -173,6 +173,12 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 				content.classList.toggle( 'minimal' );
 
 			};
+			
+			viewer.addEventListener( 'wheel', function ( event ) {
+
+				event.preventDefault();
+
+			}, { passive: false } );
```

Comment:
Um, do we actually need an event listener to prevent the wheel event? Couldn't we just remove `onmousewheel=""`?

In the documentation which has a similar structure than the example the iframe has no `onmousewheel` definition.

https://github.com/mrdoob/three.js/blob/d97939bfb112b8811d6ba8678171f0a4fa924079/docs/index.html#L45

Reply:
wow, good idea!

Reply:
@puxiao would you like to update the PR?